src/ripple/json/impl/json_value.cpp

```diff
@@ -243,6 +243,24 @@ Value::Value ( UInt value )
     value_.uint_ = value;
 }
 
+Value::Value ( std::int64_t value )
+    : type_ ( stringValue )
+    , allocated_ ( true )
+{
+    std::string str(std::to_string(value));
+    value_.string_ = valueAllocator ()->duplicateStringValue (
+        str.c_str (), str.size() );
+}
+
+Value::Value ( std::uint64_t value )
+    : type_ ( stringValue )
+    , allocated_ ( true )
+{
+    std::string str(std::to_string(value));
+    value_.string_ = valueAllocator ()->duplicateStringValue (
+        str.c_str (), str.size() );
+}
```

Comment:
Without those constructors, creating json from 64bit input won't compile (at least on gcc):

/home/mtravis/projects/rippled/src/ripple/app/misc/NetworkOPs.cpp:2392:34: error: conversion from ‘uint64_t {aka long unsigned int}’ to ‘Json::Value’ is ambiguous
     info[jss::jq_trans_overflow] = app_.overlay().getJqTransOverflow();

Referring to NetworkOPs.cpp:2392:
    info[jss::jq_trans_overflow] = app_.overlay().getJqTransOverflow();

-------------------------------------------------------------

No existing 64bit integers rendered as Json will be harmed, because none exist other than what I'm putting there.

Reply:
In [other places](https://github.com/ripple/rippled/blob/develop/src/ripple/app/misc/impl/TxQ.cpp#L1477) where we want to store 64-bit integers in JSON, we use `to_string` to make it clear that this JSON treats this differently.

I agree with @scottschurr that doing this by default has the potential for consuing semantics going forward.

Reply:
How about this? https://github.com/ximinez/rippled/commit/a51519b22dba642ad83414473be14694e447b1e2

Reply:
I've reverted the json 64bit constructor to go with the team's consensus that 64bit integers are not to be handled automatically but instead make the coder remember to make them strings. The only json change I kept is initializing the map_ object and protecting against double-free on destruction.